index.html

```diff
@@ -357,25 +357,19 @@ <h2>
             </div>
     
             
-            <form>
+            <form action="https://formspree.io/f/xnqrgeew" method="POST"> 
+                <input type="text" id="name" maxlength="30" name="name" placeholder="Yeramis" required>
+                
+                <input type="email" id="email" name="email" placeholder="natajaynj14@gmail.com" required >
+                
+                <textarea id="textarea" maxlength="500" placeholder="write your message here" required></textarea>
                 
-                <input type="text" id="name" name="name" placeholder="Yeramis">
-                <br>
-                <br>
-                <input type="email" id="email" name="email" placeholder="natajaynj14@gmail.com">
-                <br>
-                <br>
-                <textarea name="" id="textarea">  </textarea>
-                <br>
-                <br>
                 <input type="submit" value="Get in touch" id="submit" >
-                <br>
-                <br>
-                <br>
+                
               </form>
             </div>
+            <img src="img/Shape.png" alt="bottomline">
```

Comment:
- [ ] I think here you missed another image as well so please add it and fix the margin/padding for it since I can see it is very close. so please fix it 🔥🚀💯 

- Your design
![image](https://user-images.githubusercontent.com/58553711/199229515-dcbcdf15-ee8b-47c6-8e6d-199fa5739615.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/199229571-ec722fd5-0e45-47fe-b7f9-8415b29df8f9.png)

Reply:
## resolved 

- Thank you for noticing that issue too, 
- fixed it above when I put a new background image